### ISSUE TITLE
Flaps Getting Stuck (motors?)

### ISSUE THREAD

**Issue by Zuckyb** [scottbez1/splitflap#68]
I'm assembling two sets of split flaps and I've found that I'm having issues with the flaps getting stuck on the start-up rotations.
When the program starts tall the reels do a slow rotation and then a fast rotation. Some of the reels consistently get stuck around the same spots on the quick rotations. I was wondering if maybe the stepper motors don't have enough torque. If anyone has had similar problems and found solutions please let me know.

**Comment by ericyeargan** [comment]
I had a similar issue where single-stepping through flaps worked fine but the motor would stall on long rotations. I was able to resolve it by reducing the max motor speed which I accomplished by changing the MIN_PERIOD_MICROS constant in the generate_acceleration.py script from 1200 to 1650. I arrived at that new value by trial-and-error - it's the highest number I could use without the occasional stall. I then regenerated the acceleration.h header by running that script.

Hope that helps!
